openff/bespokefit/tests/conftest.py

```diff
@@ -323,10 +333,32 @@ def qc_hessian_qce_result(
             ),
             **qc_record.extras,
         },
-        properties=qc_record.properties,
+        properties=AtomicResultProperties(  # missing quadrupole moment, many fields with spaces
```

Comment:
(not blocking) Not a huge deal since it's in a test file and the tests are doing fine, but I'm adding this in case it helps someone in the future. 

```suggestion
        properties=AtomicResultProperties( 
         # missing quadrupole moment, see 
         # https://github.com/openforcefield/openff-qcsubmit/pull/195/files#diff-38e27222b4aa42cac9e280107a45496a7f27ddf6d648c7dcf330ef5622f8499aR885 
         # for breadcrumbs if needed
```